app/components/project-list-item.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+import {task} from 'ember-concurrency';
+import {inject as service} from '@ember/service';
+
+export default Component.extend({
+  store: service(),
+
+  recentBuild: computed.or('serverRecentBuild', 'localRecentBuild'),
+  serverRecentBuild: computed.readOnly('_getMostRecentBuild.lastSuccessful.value.firstObject'),
+
+  localRecentBuild: computed('project.builds', function() {
+    const allBuilds = this.get('store').peekAll('build');
+
+    const projectBuilds = allBuilds.filter(build => {
+      return build.get('project.id') === this.get('project.id');
+    });
+
+    const sortedBuilds = projectBuilds.sort((a, b) => {
```

Comment:
Could probably use sortBy
```suggestion
    const sortedBuilds = projectBuilds.sortBy('id');
```

Reply:
Can the suggestion thing do multiple lines? Dunno

Reply:
I tried using `sortBy` but it wasn't working for some reason. I'll take another whack at it.

Reply:
Oh, `sortBy` didn't work because this is an array of classes, not simple objects. So I need to `get` the ids to sort on.